classic-editor.php

```diff
@@ -4,7 +4,7 @@
  *
  * Plugin Name: Classic Editor
  * Plugin URI:  https://wordpress.org/plugins/classic-editor/
- * Description: Enables the WordPress classic editor and the old-style Edit Post screen with TinyMCE, Meta Boxes, etc. Supports the older plugins that extend this screen.

```

Comment:
This should actually be lower case I think. "Classic Editor" is a name, so needs capitalization. Here "classic editor" is a type of the editor, no capitalization.

Reply:
Also "Edit Post" no capitalization?

Reply:
"Enables the previous (classic) WordPress editor and the old-style..." is ok for me... but keep in mind that: many people confuse "classic editor" / "classic editor" / "Classic Editor" and think that they are different things, instead Classic Editor refers immediately to the plugin, and Block Editor to the new editor (codename Gutenberg).

Reply:
In short, there is a lot of confusion.

Reply:
Many people think that the "classic editor" or "editor classic" is the old WordPress editor, and the Classic Editor is the new plugin that enables the old style of editing (so a new editor with "classic editing style") but they are the same thing! Also confusing "Block Classic" of Gutenberg with old "classic editing style" or "classic editor" or "editor classic" or "Plugin Classic Editor".

Reply:
Also Visual Editor and Code Editor are confused.

Reply:
In Italy some people have wrongly classified that WordPress has 4 editors https://docs.google.com/document/d/1T_Ydg3WPhviGiXC4lD3qjfzORjhCtFCOQV_7sHEu-Rk/edit

- Code Editor (old)
- Visual Editor (old)
- Classic Editor (old)
- Gutenberg (editor in becoming)
- Block Editor (new WordPress 5.0 default editor)
- Classic Editor (plugin) new editor with old style